packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -107,6 +107,25 @@ export abstract class Package {
     this.emitTypes(context);
     code.closeFile(this.file);
 
+    if (this.types.length > 0) {
```

Comment:
Add some commentary on what is this function (and consider extracting to a separate method)